src/features/courses/course-route.ts

```diff
@@ -224,22 +225,22 @@ router.get('/questions',
                 }
 
                 // get version info - descending by startTime unless specific id is included in query
-                let versions: StudentTopicAssessmentInfo[] | undefined;
                 if (_.isNil(req.query.studentTopicAssessmentInfoId)) {
-                    versions = _.orderBy(topic.topicAssessmentInfo.studentTopicAssessmentInfo, ['startTime'],'desc');
+                    // _.orderBy puts in ascending order
+                    const versions = _.orderBy(topic.topicAssessmentInfo.studentTopicAssessmentInfo, ['startTime']).reverse();
```

Comment:
https://lodash.com/docs/4.17.15#orderBy `['
```suggestion
                    const versions = _.orderBy(topic.topicAssessmentInfo.studentTopicAssessmentInfo, ['startTime'], ['desc']);
```

Reply:
yeah, I think I was confusing _.sortBy and _.orderBy